### ISSUE TITLE
vue/require-component-is false positive

### ISSUE THREAD

**Issue by charlie0228** [nuxt/eslint-plugin-nuxt#78]
According to https://github.com/nuxt/eslint-plugin-nuxt/issues/65 and https://github.com/vuejs/eslint-plugin-vue/issues/869

I upgrade my package but still encounter the same error message.

**eslint:** 6.7.2
**eslint-plugin-nuxt:** 0.5.0
**eslint-plugin-vue:** 6.0.1
**vue-eslint-parser:** 7.0.0

```
../component.vue
  2:3  error  Expected '<component>' elements to have 'v-bind:is' attribute  vue/require-component-is
✖ 1 problem (1 error, 0 warnings)
```

Even I reinstall **eslint-plugin-nuxt** and upgrade **eslint-plugin-vue** to 6.1.2, the issue still exists.

**Comment by koalazub** [comment]
I'm still having this very same problem. I've attempted to update all dependencies and it's still returning an error. Were you able to resolve this?

**Comment by clarkdo** [comment]
Thanks for the reporting, I’ll look into this shortly

**Comment by clarkdo** [comment]
@charlie0228 @koalazub Can you run `yarn why eslint-plugin-vue` and `yarn why vue-eslint-parser` and paste result here ?

If it's possible, can you provide me a minimal repo for reproducing this issue ?

**Comment by koalazub** [comment]
> 
> 
> @charlie0228 @koalazub Can you run `yarn why eslint-plugin-vue` and `yarn why vue-eslint-parser` and paste result here ?
> 
> If it's possible, can you provide me a minimal repo for reproducing this issue ?

Didn't see this until just now. The issue was fixed when I ran npm outdated and updated the outdated packages to beta. 

These were the following packages that needed updating: 

@types/jest
@vue/eslint-config-prettier
@vue/eslint-config-typescript
@vue/test-utils
eslint
eslint-plugin-vue
typescript

I never did find the cause of the problem unfortunately, but performing the updates that were available in beta form as of this date resolved my problems.

**Comment by charlie0228** [comment]
@clarkdo @koalazub 
Minimal repo: https://github.com/charlie0228/vscode-eslint-issue/tree/vue/require-component-is
You can test in `src/pages/Example.vue` via run `npm run dev` and visit at http://localhost:8008

### yarn why eslint-plugin-vue
```
=> Found "eslint-plugin-vue@6.1.2"
info Has been hoisted to "eslint-plugin-vue"
info Reasons this module exists
   - Hoisted from "eslint-plugin-nuxt#eslint-plugin-vue"
   - Hoisted from "@nuxtjs#eslint-config-typescript#@nuxtjs#eslint-config#eslint-plugin-vue"
info Disk size without dependencies: "564KB"
info Disk size with unique dependencies: "1.46MB"
info Disk size with transitive dependencies: "7.98MB"
info Number of shared dependencies: 13
=> Found "@nuxtjs/eslint-config#eslint-plugin-vue@5.2.3"
info This module exists because "@nuxtjs#eslint-config" depends on it.
```

### yarn why vue-eslint-parser
```
=> Found "vue-eslint-parser@7.0.0"
info Has been hoisted to "vue-eslint-parser"
info Reasons this module exists
   - Hoisted from "eslint-plugin-vue#vue-eslint-parser"
   - Hoisted from "eslint-plugin-nuxt#vue-eslint-parser"
info Disk size without dependencies: "844KB"
info Disk size with unique dependencies: "6.12MB"
info Disk size with transitive dependencies: "7.34MB"
info Number of shared dependencies: 11
=> Found "@nuxtjs/eslint-config#vue-eslint-parser@5.0.0"
info Reasons this module exists
   - "@nuxtjs#eslint-config#eslint-plugin-vue" depends on it
   - Hoisted from "@nuxtjs#eslint-config#eslint-plugin-vue#vue-eslint-parser"
```

Please feel free to contact me if you need any further information.
Thanks for helping.

**Comment by clarkdo** [comment]
@charlie0228 The repo you provide works fine in my side.

The repos only contains some fixable errors which can be fixed by `npm run lint -- --fix`.

> Errors:

![image](https://user-images.githubusercontent.com/4312154/72809489-a0720300-3c53-11ea-9b6d-7fb271a67b47.png)

> Dev:

![image](https://user-images.githubusercontent.com/4312154/72809551-c13a5880-3c53-11ea-9f2f-b217c8ef2e11.png)

**Comment by charlie0228** [comment]
@clarkdo Would you remove 
`<!-- eslint-disable-next-line vue/require-component-is -->` in `src/pages/Example.vue`
and try it again?

Thanks!

**Comment by simplenotezy** [comment]
I can confirm I experience same issue

**Comment by clarkdo** [comment]
@charlie0228 @simplenotezy Can you please try to update @nuxtjs/eslint-config to v2 ?

```sh
npm install @nuxtjs/eslint-config@^2.0.0 --save-dev
```

**Comment by charlie0228** [comment]
> @charlie0228 @simplenotezy Can you please try to update @nuxtjs/eslint-config to v2 ?
> 
> ```shell
> npm install @nuxtjs/eslint-config@^2.0.0 --save-dev
> ```
> 
> Or
> 
> ```shell
> yarn upgrade @nuxtjs/eslint-config@^2.0.0
> ```

It works!
After I upgrade `@nuxtjs/eslint-config@^2.0.0` the issue is solved.
Now, my current `@nuxtjs/eslint-config` version is `2.0.2` and I also upgrade `@nuxtjs/eslint-config-typescript` to the lastest version `1.0.2`
